src/ViewModel/Converter/JobAdvertContentHeaderConverter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace eLife\Journal\ViewModel\Converter;
+
+use eLife\ApiSdk\Model\JobAdvert;
+use eLife\Patterns\ViewModel;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+final class JobAdvertContentHeaderConverter implements ViewModelConverter
+{
+    use CreatesDate;
+
+    private $urlGenerator;
+
+    public function __construct(UrlGeneratorInterface $urlGenerator)
+    {
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    /**
+     * @param JobAdvert $object
+     */
+    public function convert($object, string $viewModel = null, array $context = []) : ViewModel
+    {
+        return new ViewModel\ContentHeader(
+            $object->getTitle(),
+            null,
+            $object->getImpactStatement(),
+            true,
+            [],
+            null,
+            [],
+            [],
+            null,
+            null,
+            null,
+            null
```

Comment:
Can lose all these, as they're the default values. (Also shows we should probably rethink the `ContentHeader`'s constructor! )